contrib/bloop/src/mill/contrib/bloop/BloopImpl.scala

```diff
@@ -249,7 +250,10 @@ class BloopImpl(ev: () => Evaluator, wd: Path) extends ExternalModule { outer =>
                 else s"-Duser.dir=$wd" :: forkArgs
               }
             ),
-            mainClass = module.mainClass()
+            mainClass = module.mainClass(),
+            runtimeConfig = None,
```

Comment:
Since bloop itself [sets it to `None`](https://github.com/scalacenter/bloop/blob/266245b8ea160fd9f787b78653e227ff99450bbc/integrations/mill-bloop/src/main/scala/bloop/integrations/mill/MillBloop.scala#L63) I'm leaving it as it is.